src/bundle/Resources/views/Security/reset_user_password_success.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends '@EzPlatformAdminUi/Security/base.html.twig' %}
+
+{%- block content -%}
+    <h2 class="ez-login__header">{{ 'ezplatform.reset_user_password.change_password'|trans|desc('Change password') }}</h2>
+    <p>
+        {{ 'ezplatform.reset_user_password.success'|trans({ '%login%': path('login') })|raw|
```

Comment:
`{{ 'ezplatform.reset_user_password.success'|trans({ '%login%': path('login') })|desc('Congrats, your password has been changed, you can now <a href="%login%">login</a> to eZ Platform.')|raw }}` will it work?